routes/users.ts

```diff
@@ -0,0 +1,24 @@
+import express from "express"
+import DatabaseService from "../services/users.service"
+import { User } from "../models/user.model"
+import ErrorHandler from "../errors/errorHandler"
+
+const router = express.Router()
+let database: DatabaseService
+
+// Initialize database
+async function initializeDatabase() {
+    try {
+        database = new DatabaseService()
```

Comment:
I would rethink the implementation of this databaseservice, because it looks to me like yo uhave to initialize it in each route which will become very annoying the bigger the application becomes